CMakeLists.txt

```diff
@@ -52,9 +53,11 @@ list(APPEND DLPACK_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/contrib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/bin)
 include_directories(${DLPACK_INCLUDE_DIR})
 
-set(DLPACK_MOCK_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/contrib/mock_main.cc
-                    ${CMAKE_CURRENT_SOURCE_DIR}/contrib/mock_c.c)
-add_executable(mock ${DLPACK_MOCK_SRCS})
+if(BUILD_MOCK)
+  set(DLPACK_MOCK_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/contrib/mock_main.cc
```

Comment:
Fix building with Visual Studio generator. As far as I can see `mock_main.cc` is not designed to be used on Windows:
- https://github.com/dmlc/dlpack/blob/5c792cef3aee54ad8b7000111c9dc1797f327b59/contrib/mock_main.cc#L3
- https://github.com/dmlc/dlpack/blob/5c792cef3aee54ad8b7000111c9dc1797f327b59/contrib/dlpack/dlpackcpp.h#L50